### PR TITLE
Rename and update Sampler Advertisement man page

### PR DESCRIPTION
**Pull request by nichamon**

### CODE REVIEW

**Comment by tom95858** [comment]
@nichamon, these really aren't sampler advertisements...correct? They are ldmsd peer advertisements. The ldmsd peer may have no samplers at all.

File: ldms/man/ldmsd_sampler_advertisement.man [hunk 1]
```diff
@@ -47,7 +47,7 @@ ldmsd_sampler_advertisement - Manual for LDMSD Sampler Advertisement
 
 .SH DESCRIPTION
 
-LDMSD Sampler Discovery is a capability that enables LDMSD automatically add
+LDMSD Sampler Advertisement is a capability that enables LDMSD automatically add
```
Comment (tom95858) [inline ldms/man/ldmsd_sampler_advertisement.man:50]: LDMSD Peer Daemon advertisement is a capability that enables LDMSD to automatically add

File: ldms/man/ldmsd_sampler_advertisement.man [hunk 2]
```diff
@@ -47,7 +47,7 @@ ldmsd_sampler_advertisement - Manual for LDMSD Sampler Advertisement
...
 producers that its hostname matches a given regular expression. The feature
```
Comment (tom95858) [inline ldms/man/ldmsd_sampler_advertisement.man:51]: producers for advertisers whose hostname matches a regular expression.

File: ldms/man/ldmsd_sampler_advertisement.man [hunk 3]
```diff
@@ -47,7 +47,7 @@ ldmsd_sampler_advertisement - Manual for LDMSD Sampler Advertisement
...
 eliminates the need for manual configuration of sampler hostname in the
```
Comment (tom95858) [inline ldms/man/ldmsd_sampler_advertisement.man:52]: reduces the need for the manual configuration of producers in configuration files

**Review by tom95858 — CHANGES_REQUESTED** [review]
@nichamon ... should we change the man file name?

**Comment by nichamon** [comment]
> @nichamon ... should we change the man file name?

Yes, I agree.

**Comment by tom95858** [comment]
@nichamon is this ready to go?

**Pull request merged**